src/css/flags/time-slider-above.less

```diff
@@ -286,9 +280,13 @@
       .jw-progress,
       .jw-buffer,
       .jw-knob {
-        background: none;
         border: none;
         box-shadow: none;
+        border-radius: 0;
```

Comment:
Why 'undo' this? Shouldn't radius only be set in skin mixins and seven styles only?